lib/routes.js

```diff
@@ -24,7 +24,8 @@ const defaults = {
 		national:    true,
 		regionalExp: true,
 		regional:    true
-	}
+	},
+	bahncard: 		0 //25_1: 1, 25_2: 2, 50_1: 3, 50_2: 4
```

Comment:
What is the difference between `25_1` and `25_2`? 1st class and 2nd class?

Would like to have more intuitive values here.